zerver/openapi/zulip.yaml

```diff
@@ -1910,6 +1910,36 @@ paths:
                         "msg": "User not authorized for this query",
                         "result": "error"
                     }
+  /streams/{stream_id}:
+    delete:
+      description: Delete the stream.
+      parameters:
+      - name: stream_id
+        in: path
+        description: Delete the particular stream.
```

Comment:
This is a description for the parameter, so this should be `The ID of the stream to be deleted.`